rpcs3/Emu/Memory/vm.cpp

```diff
@@ -805,11 +805,11 @@ namespace vm
 			vm::writer_lock lock(0);
 
 			// Deallocate all memory
-			for (auto it = m_map.begin(), end = m_map.end(); !m_common && it != end;)
+			for (auto it = m_map.begin(), end = m_map.end(); it != end;)
 			{
 				const auto next = std::next(it);
 				const auto size = it->second.first;
-				_page_unmap(it->first, size, it->second.second.get());
+				_page_unmap(it->first, size, !m_common ? it->second.second.get() : nullptr);
```

Comment:
I'm not sure what are you trying to signify by these changes, second.get() should already be nullptr in these cases.

Reply:
I see.

Reply:
Fixed.